src/bridge/Makefile.am

```diff
@@ -277,24 +239,35 @@ polkitdir        = $(datadir)/polkit-1/actions
 polkit_DATA = $(polkit_in_files:.policy.in=.policy)
 
 %.policy: %.policy.in $(PO_FILES)
-	GETTEXTDATADIRS=$(srcdir)/po msgfmt --xml -d $(top_srcdir)/po --template $< --output $@
+	$(AM_V_GEN) GETTEXTDATADIRS=$(srcdir)/po msgfmt --xml -d $(top_srcdir)/po --template $< --output $@
 
 CLEANFILES += $(polkit_DATA)
 endif
 
 # -----------------------------------------------------------------------------
 # PCP
 
-# make sure this ends up in the tarball, even with --disable-pcp
-pmlogconf_files = src/bridge/pmlogconf/cockpit
-EXTRA_DIST += $(pmlogconf_files)
-
 if ENABLE_PCP
 
+pmlogconfdir = $(localstatedir)/lib/pcp/config/pmlogconf/tools
+dist_pmlogconf_DATA = src/bridge/pmlogconf/cockpit
+
```

Comment:
I still think we want the file in tarballs even with `--disable-pcp`, as the deleted comment says. Does the `_DATA` rule work differently?

Reply:
Thanks for raising this.  I thought it did, but by reading the generated code in `Makefile.in`, I think that maybe I was wrong.  I'll change this back.

Reply:
Ah, no.  I was right in the first place.  We get this emitted:

```
@ENABLE_PCP_TRUE@dist_pmlogconf_DATA = src/bridge/pmlogconf/cockpit
```

but also this:

```
am__dist_pmlogconf_DATA_DIST = src/bridge/pmlogconf/cockpit
...
DIST_COMMON = ... $(am__dist_pmlogconf_DATA_DIST) ...
```

so, indeed, this is exactly the point of using `dist_*` instead of `EXTRA_DIST`.

Reply:
Ack, thanks for checking!